### PR TITLE
Report - Yield Performance Report (FE)

### PR DESCRIPTION
**Pull request by callum-r-young**
### Change description ###
Added Yield Performance Report.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: server/routes/reporting/standard-report/standard-report-print.js [hunk 1]
```diff
@@ -49,6 +49,10 @@ async function standardReportPrint(app, req, res, reportKey, data) {
           text = text === 'Civ' ? 'Civil' : 'Criminal';
         }
 
+        if (header.id === 'comments') {
+          text = text.replace('\n','\n\n')
```
Comment (ricardofreitasrocha) [inline server/routes/reporting/standard-report/standard-report-print.js:53]: should this not be `replaceAll()` or use global regex `/\n/g`? could be that we need to replace more than one instance of `\n` for multiple groups of comments?

File: server/routes/reporting/standard-report/standard-report.controller.js [hunk 2]
```diff
@@ -363,6 +363,10 @@
             output = `${capitalise(output.split('-')[0])} - ${output.split('-')[1]}`;
           }
 
+          if (header.id === 'comments') {
+            output = output.replace('\n','<br><br>')
```
Comment (ricardofreitasrocha) [inline server/routes/reporting/standard-report/standard-report.controller.js:367]: same as above

**Review by ricardofreitasrocha — CHANGES_REQUESTED** [review]

File: server/routes/reporting/standard-report/standard-report-print.js (hunk 1 above)
Comment (callum-r-young) [reply server/routes/reporting/standard-report/standard-report-print.js]: you're right. updated.

File: server/routes/reporting/standard-report/standard-report.controller.js (hunk 2 above)
Comment (callum-r-young) [reply server/routes/reporting/standard-report/standard-report.controller.js]: updated.

**Review by ricardofreitasrocha — APPROVED** [review]

**Pull request merged**